src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -295,6 +295,12 @@ private static Option2<CodeStyleOption2<AddImportPlacement>> CreateUsingDirectiv
                 EditorConfigStorageLocation.ForBoolCodeStyleOption("csharp_style_allow_blank_line_after_colon_in_constructor_initializer_experimental", CodeStyleOptions2.TrueWithSilentEnforcement),
                 new RoamingProfileStorageLocation($"TextEditor.CSharp.Specific.{nameof(AllowBlankLineAfterColonInConstructorInitializer)}")});
 
+        public static readonly Option2<CodeStyleOption2<bool>> PreferTrailingCommas = CreateOption(
+            CSharpCodeStyleOptionGroups.ExpressionLevelPreferences, nameof(PreferTrailingCommas),
+            defaultValue: s_trueWithSuggestionEnforcement,
```

Comment:
Should this be true or false?

Reply:
this should definitely be false.

Reply:
and it should be silent.

Reply:
I would expect true/silent.

Reply:
i'm somewhat amenable to that..  i do think 'silent' has to be case here.

Reply:
true/silent makes the most sense.

Although, I do think it improves readability (for me personally), in that it helps clarify it is an existentially qualified collection. My mind reads opening line `{` with a series of lines ending in  `,` followed by a matching `}` as one giant math expression "construct collection with these members..."